MAUI/DataGrid/pull-to-refresh.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: post
+title: PullToRefresh in .NET MAUI DataGrid control | Syncfusion
+description: Learn here all about PullToRefresh support in Syncfusion .NET MAUI DataGrid (SfDataGrid) control and more.
+platform: MAUI
+control: SfDataGrid
+documentation: UG
+---
+
+# PullToRefresh in MAUI DataGrid (SfDataGrid)
+
+PullToRefresh is a refresh control that allows you to interact and refresh the view loaded in it. The PullToRefresh control allows you to refresh the view upon performing the pull-to-refresh action. A progress indicator will be shown while starting the pulling. The application will be refreshed once you have pulled down a certain distance and released the touch.
```

Comment:
The above description is not required. Check and remove.

Reply:
![image](https://github.com/syncfusion-content/maui-docs/assets/113881396/ed04398b-c270-48af-b559-876185ec09f8)
Had removed as discussed.